ci/tasks/prepare-e2e-codebuild.yml

```diff
@@ -0,0 +1,68 @@
+---
+platform: linux
+inputs:
+  - name: pay-ci
+outputs:
+  - name: run-codebuild-configuration
+params:
+  CODEBUILD_PROJECT_NAME:
+  CODEBUILD_SOURCES_BUCKET:
+  PROJECT_UNDER_TEST:
+  RELEASE_TAG_UNDER_TEST:
+  AWS_ACCESS_KEY_ID:
+  AWS_SECRET_ACCESS_KEY:
+  AWS_SESSION_TOKEN:
+run:
+  path: /bin/bash
+  args:
+    - -euo
+    - pipefail
+    - -c
+    - |
+      echo "Installing zip"
+      apk add -q --no-progress zip
+      echo "Done"
+      echo
+
+      pushd "pay-ci" >> /dev/null
+      echo "Zipping pay-ci"
```

Comment:
What's are the contents of this `pay-ci` directory?

Reply:
The whole pay-ci directory

Reply:
Is it the whole repo? Or just the endtoend folder?

Reply:
Why does codebuild need the whole pay-ci repo?

Reply:
If we do change this I'd like to merge everything as-is and recode later, there's a lot of moving parts at the moment and this change would mean changing something in all of them

Reply:
Understood 👍